server/src/test/java/org/opensearch/search/aggregations/bucket/terms/IncludeExcludeTests.java

```diff
@@ -297,4 +298,159 @@ private IncludeExclude serializeMixedRegex(IncludeExclude incExc) throws IOExcep
         }
     }
 
+    private static BytesRef[] toBytesRefArray(String... values) {
+        BytesRef[] bytesRefs = new BytesRef[values.length];
+        for (int i = 0; i < values.length; i++) {
+            bytesRefs[i] = new BytesRef(values[i]);
+        }
+        return bytesRefs;
+    }
+
+    public void testPrefixOrds() throws IOException {
```

Comment:
should we add small tests for only include / excludes cases, theres a few code paths specific to those cases.

Reply:
Done! I added an exclude-only test, then flipped it to include-only and confirmed that the resulting bits were the opposite.